doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -156,23 +156,69 @@ The below set of steps define the compliance, verification and certification pro
+toolchains thanks to a few low constraints
+[quickly achievable](https://www.sdxcentral.com/articles/news/opnfvs-6th-release-brings-testing-capabilities-that-orange-is-already-using/2018/05/).
+
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) is a more recent
+project which leverages on the common test case execution proposed by Xtesting.
+Thanks to a simple test case list, this tool deploys anywhere plug-and-play
+[CI/CD toolchains in a few commands](https://wiki.opnfv.org/pages/viewpage.action?pageId=32015004).
+In addition of this teaching capability needed by the Network Automation
+journey, it supports multiple components such as Jenkins and Gitlab CI (test
+schedulers) and
+[multiple deployment models](https://lists.opnfv.org/g/opnfv-tsc/message/5702)
+such as all-in-one or centralized services.
+
+This 2 projects combined meet the CNTT requirements about verification,
+compliance and certification:
+- smoothly assemble multiple heterogeneous test cases
+- generate the Jenkins jobs in
+  [OPNFV Releng](https://git.opnfv.org/releng/tree/jjb/airship/cntt.yaml) to
+  verify CNTT RI
+- deploy local CI/CD toolchains everywhere to check compliance with CNTT
+- [dump all test case results and logs](http://artifacts.opnfv.org/functest/9ID39XK47PMZ.zip)
+  for third-party certification review
+
+All test cases must conform with Xtesting to ensure the smooth integration in
+the CI toolchain as expected by CNTT.
+
+It's worth mentioning that Functest already leverages on Xtesting. Here are the
+issues tracking the updates of the existing OPNFV test projects:
+- Bottlenecks: https://github.com/cntt-n/CNTT/issues/510
+- NFVBench: https://github.com/cntt-n/CNTT/issues/865
+- StorPerf: https://github.com/cntt-n/CNTT/issues/673
+- VSPERF: https://github.com/cntt-n/CNTT/issues/511
+- YardStick: https://github.com/cntt-n/CNTT/issues/509
 
 <a name="2.7.1"></a>
-### 2.7.1 Best Practices (General)
-The NFVI certification framework will be guided by the following core principles:
--   Implementing, and adhering to, Standardized Test Methodology / flow, Test Plan, and Test Case Suites, which promotes scalability using repeatable processes.
--   Integration with Automated Tool-Chains, such as XTesting or Dovetail, for continuous deployment, validation, and centralization of test harnesses and results visualization.
-- Alliance and execution of OVP flows and methodologies, which supports common structures for code, artifact generation and repository, certification criteria, etc.)
--   Where possible, leveraging ONAP Network and Service Models, with identified VNF-specific parameters
--   Utilizing Standard certification criteria.
--   Defining CNTT reference architecture (RA) as scenarios, and having all test cases for the RA be involved in OVP (could also be addressed in OVP as CNTT test)
--   Add test cases from operators, which operators already tested in their environment
```

Comment:
Keep, as this is general / generic guidance and principles for Compliance "Frameworks".

Reply:
revert even if I disagree here about the framework structure.